package.json

```diff
@@ -8,8 +8,9 @@
   "license": "MIT",
   "scripts": {
     "start": "webpack serve --open",
-    "test": "cypress open",
-    "ci": "yarn lint && webpack serve & (wait-on http://localhost:7890 && cypress run)",
+    "test:integration": "cypress open",
+    "test": "jest",
+    "ci": "yarn lint && yarn test && webpack serve & (wait-on http://localhost:7890 && cypress run)",
```

Comment:
Possibly this could be reworked slightly. `Test` could run all the test, and there could be specific integration and unit test scripts.

Reply:
Yes, that's fair! I did think about `test:unit`.